public/icons/remove.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="28" height="28" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
+<circle cx="14" cy="14" r="14" fill="#DB1B1B"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M14.8776 14.0131L19.51 9.28938C19.7093 9.07197 19.7004 8.73571 19.4899 8.52913C19.2794 8.32255 18.943 8.32001 18.7294 8.52338L14.1043 13.2398L9.28955 8.51608C9.1517 8.37129 8.94587 8.31321 8.75267 8.36458C8.55948 8.41596 8.40968 8.56861 8.36196 8.76274C8.31424 8.95688 8.3762 9.16159 8.52356 9.29668L13.331 14.0131L8.52356 18.7295C8.3202 18.9431 8.32274 19.2794 8.52931 19.49C8.73589 19.7005 9.07214 19.7094 9.28955 19.5101L14.1043 14.7864L18.7294 19.5028C18.943 19.7061 19.2794 19.7036 19.4899 19.497C19.7004 19.2904 19.7093 18.9542 19.51 18.7368L14.8776 14.0131Z" fill="white"/>
+</svg>
```

Comment:
@zacck please use Elm to use SVG. you can see an example here: https://github.com/cambiatus/frontend/blob/feature/new-topbar/src/elm/Icons.elm#L15

Reply:
@lucca65  done